datasets/sem_eval_2020_task_11/sem_eval_2020_task_11.py

```diff
@@ -0,0 +1,215 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import glob
+import os
+
+import datasets
+
+
+_CITATION = """
+@misc{martino2020semeval2020,
+      title={SemEval-2020 Task 11: Detection of Propaganda Techniques in News Articles},
+      author={G. Da San Martino and A. Barrón-Cedeño and H. Wachsmuth and R. Petrov and P. Nakov},
+      year={2020},
+      eprint={2009.02696},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """\
+Propagandistic news articles use specific techniques to convey their message,
+such as whataboutism, red Herring, and name calling, among many others.
+The Propaganda Techniques Corpus (PTC) allows to study automatic algorithms to
+detect them. We provide a permanent leaderboard to allow researchers both to
+advertise their progress and to be up-to-speed with the state of the art on the
+tasks offered (see below for a definition).
+"""
+
+_HOMEPAGE = "https://propaganda.qcri.org/ptc/index.html"
+
+_LICENSE = ""
+
+
+class SemEval2020Task11(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class SemEval_2020Task_11(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `sem_eval_2020_task_11`